enterprise/server/workflow/workflow.go

```diff
@@ -308,6 +308,11 @@ func (ws *workflowService) createActionForWorkflow(ctx context.Context, wf *tabl
 			"--trigger_event=" + wd.EventName,
 			"--trigger_branch=" + wd.TargetBranch,
 		},
+		Platform: &repb.Platform{
+			Properties: []*repb.Platform_Property{
+				{Name: "container-image", Value: "docker://gcr.io/flame-public/buildbuddy-ci-runner:v1.7.0"},
```

Comment:
Haven't pushed this image to the container registry yet -- does this proposed name & tag look fine? I just set the version to match the latest BB release.

Reply:
this looks fine to me - just pinged you on slack but I was wondering if it'd be easier to just use the executor image? Fewer images to keep up to date would be nice.